src/Microsoft.DotNet.ProjectJsonMigration/PackageConstants.cs

```diff
@@ -26,55 +26,1810 @@ internal class PackageConstants
         public const string DotnetTestXunit = "dotnet-test-xunit";
         public const string DotnetTestMSTest = "dotnet-test-mstest";
 
-        public static readonly IDictionary<string, PackageDependencyInfo> ProjectDependencyPackages = 
-            new Dictionary<string, PackageDependencyInfo> {
-                {"Microsoft.EntityFrameworkCore.Tools", new PackageDependencyInfo {
-                    Name = "Microsoft.EntityFrameworkCore.Tools",
-                    Version = ConstantPackageVersions.AspNetToolsVersion } },
-                { "Microsoft.AspNetCore.Razor.Tools", null },
-                { "Microsoft.AspNetCore.Razor.Design", null },
-                { "Microsoft.VisualStudio.Web.CodeGenerators.Mvc", new PackageDependencyInfo {
-                    Name = "Microsoft.VisualStudio.Web.CodeGeneration.Design",
-                    Version = ConstantPackageVersions.AspNetToolsVersion } },
-                { "Microsoft.VisualStudio.Web.CodeGeneration.Tools", null},
-                { TestSdkPackageName, new PackageDependencyInfo {
-                    Name = TestSdkPackageName,
-                    Version = ConstantPackageVersions.TestSdkPackageVersion } },
-                { XUnitPackageName, new PackageDependencyInfo {
-                    Name = XUnitPackageName,
-                    Version = ConstantPackageVersions.XUnitPackageVersion } },
-                { XUnitRunnerPackageName, new PackageDependencyInfo {
-                    Name = XUnitRunnerPackageName,
-                    Version = ConstantPackageVersions.XUnitRunnerPackageVersion } },
-                { MstestTestAdapterName, new PackageDependencyInfo {
-                    Name = MstestTestAdapterName,
-                    Version = ConstantPackageVersions.MstestTestAdapterVersion } },
-                { MstestTestFrameworkName, new PackageDependencyInfo {
-                    Name = MstestTestFrameworkName,
-                    Version = ConstantPackageVersions.MstestTestFrameworkVersion } },
-                { DotnetTestXunit, null },
-                { DotnetTestMSTest, null },
+        public static readonly IDictionary<PackageDependencyInfo, PackageDependencyInfo> ProjectDependencyPackages =
+            new Dictionary<PackageDependencyInfo, PackageDependencyInfo> {
+                {
+                    new PackageDependencyInfo {
+                        Name = "Microsoft.EntityFrameworkCore.Tools",
```

Comment:
True. Is there a use case for extension without a recompile? What's the story for distributing the data file, reading it from a well-known path, etc? Would this be something users would use? Since configurability adds fragility, we should consider whether there's a good use case for it.